libs/cases/frontend/view/src/lib/components/cases-brows/case-brows.component.html

```diff
@@ -0,0 +1,44 @@
+<ix-content>
+	<ix-button [routerLink]="['/cases/open']" class="absolute top-0 left-4">
+		View Open Cases
+	</ix-button>
+	<ix-button [routerLink]="['/cases/create']" class="absolute top-0 right-4">
+		Create Case
+	</ix-button>
+
+	<table class="table">
+		<thead>
+			<tr>
+				<th scope="col">#</th>
+				<th scope="col">Title</th>
+				<th scope="col">Priority</th>
+				<th scope="col">Status</th>
+				<th scope="col">Assigned to</th>
+				<th scope="col">Due to</th>
+				<th scope="col">Description</th>
+			</tr>
+		</thead>
+        <tbody>
+            @for(_case of sortedCases; track _case.handle) {
+                <tr [routerLink]="['/cases', _case.handle]" [ngClass]="{
+              'emergency': _case.priority === 'EMERGENCY',
+              'status-open': _case.status === 'OPEN',
+              'status-inprogress': _case.status === 'INPROGRESS',
+              'status-overdue': _case.status === 'OVERDUE',
+              'status-onhold': _case.status === 'ONHOLD',
+              'status-done': _case.status === 'DONE',
+              'status-cancelled': _case.status === 'CANCELLED',
+              'status-archived': _case.status === 'ARCHIVED'
+            }">
+                    <th scope="row">{{ _case.handle }}</th>
+                    <td>{{ _case.title }}</td>
+                    <td>{{ _case.priority }}</td>
+                    <td>{{ _case.status }}</td>
+                    <td>{{ _case.assignedTo }}</td>
+                    <td>{{ _case.dueDate }}</td>
+                    <td>{{ _case.description }}</td>
+                </tr>
+            }
```

Comment:
Logic here is quite dangerous maybe it would be better encapsulated in a small component?

Reply:
I don't know how but I can try

Reply:
did you try?

Reply:
Yeah I gave it a try. I tried to create a different component under shared but had trouble referencing it.

Reply:
Thats okay! it was a suggestion no must have :)